csharp/test/Ice/dictMapping/MyClassAMDI.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace ZeroC.Ice.Test.DictMapping
 {
-    public sealed class MyClassAsync : IMyClassAsync
+    public sealed class AsyncMyClass : IAsyncMyClass
```

Comment:
We need to rename the file names at some point (for another PR?).

Reply:
Yes, the file names definitely need to be fixed in all the tests. I thought we had an issue open for doing this one day... Apparently not. I'll open one.
But I think this is best left for another PR, since it'll require modifying the test files across all the language mappings for consistency.